test/ranges.jl

```diff
@@ -887,7 +887,15 @@ function range_fuzztests(::Type{T}, niter, nrange) where {T}
         @test m == length(r)
         @test strt == first(r)
         @test Δ == step(r)
-        @test stop ≈ last(r) atol = eps((n-1)*Δ) + eps(stop) # account for intermediate rounding in computation of stop
+        # potential floating point error:
+        #   stop = strt + (n-1)*Δ
+        #      *        => eps((n-1)*Δ)/2
+        #      +        => eps(stop)/2
+        #   last(r)
+        #     rat(strt) => eps(strt)/2
+        #     rat(Δ)    => (n-1)*eps(Δ)/2
+        #     T(...)    => eps(last(r))/2
+        @test stop ≈ last(r) atol = eps((n-1)*Δ)/2 + eps(stop)/2 + eps(strt)/2 + (n-1)*eps(Δ)/2 + eps(last(r))/2
```

Comment:
I think this tolerance can be simplified significantly.

Reply:
If you wanted a weaker bound, you can use `eps(x) <= eps(T) * abs(x)` and so you can have:
```suggestion
        @test stop ≈ last(r) atol = (abs(strt)/2 + (n-1)*abs(Δ) + abs(stop)) * eps(T)
```
is that what you had in mind?

Reply:
yeah, that looks a lot better.